gravitee-apim-rest-api/gravitee-apim-rest-api-management/gravitee-apim-rest-api-management-rest/src/main/java/io/gravitee/rest/api/management/rest/resource/ApiResource.java

```diff
@@ -898,6 +893,26 @@ public Response promoteAPI(@RequestBody @Valid @NotNull final PromotionRequestEn
         return Response.ok(promotionService.promote(GraviteeContext.getExecutionContext(), promotion.getId())).build();
     }
 
+    @POST
```

Comment:
I would have added `apiId` in `VeryApiParam` class and used the `verify` resource of `ApisResource` class. WDYT ?

Reply:
ok done